src/System.Private.TypeLoader/src/Internal/TypeSystem/RuntimeNoMetadataType.cs

```diff
@@ -280,6 +283,69 @@ public override TypeDesc UnderlyingType
             }
         }
 
+        private void GetTypeNameHelper(out string name, out string nsName, out string assemblyName)
+        {
+            TypeReferenceHandle typeRefHandle;
+            QTypeDefinition qTypeDefinition;
+            MetadataReader reader;
+
+            RuntimeTypeHandle genericDefinitionHandle = GetTypeDefinition().GetRuntimeTypeHandle();
+            Debug.Assert(!genericDefinitionHandle.IsNull());
+
+            // NOTE: We're ignoring the containing type for compatiblity with SigFormat.cpp
```

Comment:
I saw in other places that we also ignore the containing type name, and with that same exact comment. I don't know the full context behind it, but with some experiments with the Desktop CLR, the containing type name is also omitted, so I copied over the comment the way it was.